Cargo.toml

```diff
@@ -64,46 +64,46 @@ wynd-utils = "0.4.1"
+dao-proposal-single = { path = "./contracts/proposal/dao-proposal-single", version = "2.1.5" }
+dao-proposal-sudo = { path = "./test-contracts/dao-proposal-sudo", version = "2.1.5" }
+dao-testing = { path = "./packages/dao-testing", version = "2.1.5" }
+dao-vote-hooks = { path = "./packages/dao-vote-hooks", version = "2.1.5" }
+dao-voting = { path = "./packages/dao-voting", version = "2.1.5" }
+dao-voting-cw20-balance = { path = "./test-contracts/dao-voting-cw20-balance", version = "2.1.5" }
+dao-voting-cw20-staked = { path = "./contracts/voting/dao-voting-cw20-staked", version = "2.1.5" }
+dao-voting-cw4 = { path = "./contracts/voting/dao-voting-cw4", version = "2.1.5" }
+dao-voting-cw721-staked = { path = "./contracts/voting/dao-voting-cw721-staked", version = "2.1.5" }
+dao-voting-native-staked = { path = "./contracts/voting/dao-voting-native-staked", version = "2.1.5" }
 
 # v1 dependencies. used for state migrations.
-cw-core-v1 = { package = "cw-core", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-cw-proposal-single-v1 = { package = "cw-proposal-single", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
+cw-core-v1 = { package = "cw-core", version = "0.1.0" }
+cw-proposal-single-v1 = { package = "cw-proposal-single", version = "0.1.0" }
 cw-utils-v1 = {package = "cw-utils", version = "0.13"}
-cw20-stake-external-rewards-v1 = { package = "stake-cw20-external-rewards", version = "0.2.6", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-cw20-stake-reward-distributor-v1 = { package = "stake-cw20-reward-distributor", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-cw20-stake-v1 = { package = "cw20-stake", version = "0.2.6", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-cw20-staked-balance-voting-v1 = { package = "cw20-staked-balance-voting", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-cw4-voting-v1 = { package = "cw4-voting", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
-voting-v1 = { package = "voting", version = "0.1.0", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v1.0.0" }
+cw20-stake-external-rewards-v1 = { package = "stake-cw20-external-rewards", version = "0.2.6" }
+cw20-stake-reward-distributor-v1 = { package = "stake-cw20-reward-distributor", version = "0.1.0" }
+cw20-stake-v1 = { package = "cw20-stake", version = "0.2.6" }
+cw20-staked-balance-voting-v1 = { package = "cw20-staked-balance-voting", version = "0.1.0" }
+cw4-voting-v1 = { package = "cw4-voting", version = "0.1.0" }
+voting-v1 = { package = "dao-voting", version = "0.1.0" }
 stake-cw20-v03 = { package = "stake-cw20", version = "0.2.6", git = "https://github.com/DA0-DA0/dao-contracts.git", tag = "v0.3.0" }
```

Comment:
Publish legacy contract and remove git reference?

Reply:
Done!